tests/TestHelper.php

```diff
@@ -107,4 +108,18 @@ public static function formatRule($rule)
 
         return $rule;
     }
+
+    public static function retry($function, $delay, $maxRetries)
```

Comment:
Since the `delay` and `maxRetries` are almost always the same in our test suite, what do you think of adding `100000` and `40` as the default values? This makes the code we use to retry more readable IMO, because we don't have to pass meaningless (at first sight) values like this. WDYT?
```suggestion
    public static function retry($function, $delay = 100000, $maxRetries = 40)
```